x-pack/plugins/canvas/public/components/export_app/export_app.component.tsx

```diff
@@ -31,9 +31,7 @@ export const ExportApp: FC<Props> = ({ workpad, selectedPageIndex, initializeWor
     <div className="canvasExport" data-shared-page={selectedPageIndex + 1}>
       <div className="canvasExport__stage">
         <div className="canvasLayout__stageHeader">
-          <Link name="loadWorkpad" params={{ id }}>
-            Edit Workpad
-          </Link>
+          <RoutingLink to={`/workpad/${id}`}>Edit Workpad</RoutingLink>
```

Comment:
i18n?

Reply:
This is on the PDF export page, so I don't think it really needs i18n since it's never actually going to be seen.  Not sure why that link is even there really.